config/hltweaker/client.toml

```diff
@@ -0,0 +1,22 @@
+#Show tool harvest level in tooltips. NEVER = never shows, ADVANCED = shows when shift is held or advanced tooltips are enabled, ALWAYS = always shows.
+#Allowed Values: NEVER, ADVANCED, ALWAYS
+tool_harvest_levels_display_mode = "NEVER"
```

Comment:
You would want to hide harvest tooltip because other mods doesn't support this mod.
For example, Terramity ores require diamond pickaxe while they actually require their own tier